browser/registry.go

```diff
@@ -90,3 +91,27 @@ func (r *remoteRegistry) isRemoteBrowser() (string, bool) {
 
 	return wsURL, true
 }
+
+// browserRegistry stores browser instances indexed per
+// iteration as identified by VUID-scenario-iterationID.
+type browserRegistry struct {
```

Comment:
The reason we need the `browserRegistry` in the `RootModule` is because there's no other way to store the references to `browser` and `browserProcess` at the iteration scope, so that we can later retrieve them and do the necessary cleanup, correct?

Reply:
Correct, the necessary cleanup, and be able to reuse it for multiple `browser` method calls in the same iteration.